packages/connect/src/device/__tests__/checkFirmwareRevision.test.ts

```diff
@@ -102,13 +102,17 @@ describe(checkFirmwareRevision.name, () => {
             expected: { success: false, error: 'cannot-perform-check-offline' },
         },
         {
-            it: "doesn't check anything for new versions with security chip (not implemented yet)",
+            it: 'passes also for a different device',
```

Comment:
```suggestion
            it: 'passes also for a T3T1 device',
```

\+ maybe I would event consider to adding some test to test all know devices with a defined map of all devices (so Typescript force you to run such test for every device from `DeviceModelInternal` enum

Reply:
Done in new commit 6e294b875e946256e44f9cc74407ff9dc1f2c5a8.

We could list all devices manually here, and have it exhaustively typed so TS forces me to maintain it - that would work, but I propose a more straightforward way: simply iterate the test foreach `DeviceModelInternal` value.